src/screens/app/Components/ProfileImageList/ProfileImageList.js

```diff
@@ -0,0 +1,415 @@
+        },
+        bookmarkers: {
+          dennyliang: true,
+        },
+        comments: [
+          {
+            userName: 'Model1234',
+            userImage:
+              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+            comment: 'Slay girllll!!!',
+          },
+        ],
+        img:
+          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+        date_posted: '10-10-2019',
+      },
+      {
+        post_id: 9,
+        isLiked: false,
+        isBookmarked: false,
+        commentsLength: 1,
+        caption: 'Quarantine selfie #corona',
+        likes: 2,
+        saves: 0,
+        username: 'bujarsefa',
+        likers: {
+          dennyliang: true,
+        },
+        bookmarkers: {
+          dennyliang: true,
+        },
+        comments: [
+          {
+            userName: 'Model1234',
+            userImage:
+              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+            comment: 'Slay girllll!!!',
+          },
+        ],
+        img:
+          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+        date_posted: '10-10-2019',
+      },
+      {
+        post_id: 10,
+        isLiked: false,
+        isBookmarked: false,
+        commentsLength: 1,
+        caption: 'Quarantine selfie #corona',
+        likes: 2,
+        saves: 0,
+        username: 'bujarsefa',
+        likers: {
+          dennyliang: true,
+        },
+        bookmarkers: {
+          dennyliang: true,
+        },
+        comments: [
+          {
+            userName: 'Model1234',
+            userImage:
+              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+            comment: 'Slay girllll!!!',
+          },
+        ],
+        img:
+          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
+        date_posted: '10-10-2019',
+      },
+      {
+        post_id: 11,
+        isLiked: false,
+        isBookmarked: false,
+        commentsLength: 1,
+        caption: 'Quarantine selfie #corona',
+        likes: 2,
+        saves: 0,
+        username: 'bujarsefa',
+        likers: {
+          dennyliang: true,
+        },
+        bookmarkers: {
+          dennyliang: true,
+        },
+        comments: [
+          {
+            userName: 'Model1234',
+            userImage:
+              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+            comment: 'Slay girllll!!!',
+          },
+        ],
+        img:
+          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
```

Comment:
Should be pulling from the database now, pls make sure the code is aligned with kartikeya and dennys new code